### ISSUE TITLE
Error installing 1.2.10 release in Ubuntu 21.04 when checking xmldom package integrity

### ISSUE THREAD

**Issue by luizvital** [Chia-Network/chia-blockchain-gui#569]
Where goes the output

```
$ sh install-gui.sh 
apt-get is /usr/bin/apt-get
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
libxss1 is already the newest version (1:1.2.3-1).
nodejs is already the newest version (12.21.0~dfsg-3ubuntu1).
npm is already the newest version (7.5.2+ds-1).
0 upgraded, 0 newly installed, 0 to remove and 4 not upgraded.

Installing on Ubuntu 20.04 LTS or newer: Using installed node.js version.
Running git submodule update --init --recursive.

Running git submodule update.

npm WARN deprecated querystring@0.2.0: The querystring API is considered Legacy. new code should use the URLSearchParams API instead.
npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm WARN deprecated messageformat@2.3.0: Package renamed as '@messageformat/core', see messageformat.github.io for more details. 'messageformat' will eventually provide a polyfill for Intl.MessageFormat, once it's been defined by Unicode & ECMA.
npm WARN deprecated svgo@1.3.2: This SVGO version is no longer supported. Upgrade to v2.x.x.
npm WARN tarball tarball data for electron-osx-sign@git+ssh://git@github.com/electron/electron-osx-sign.git#02d6ee7bbe4a7f6a2435baef7db84ced5dbc7990 (sha512-4rg0ghhKXHGrSBsE0tlrpNIENbkd0s777WFPeRCySl6pVmwA69Wcmzr79WG2gzGwNzSkiWOhdIoVV/4dBE+6Jw==) seems to be corrupted. Trying again.
npm WARN tarball tarball data for xmldom@git+ssh://git@github.com/xmldom/xmldom.git#c568938641cc1f121cef5b4df80fcfda1e489b6e (sha512-XKfWBSJe4BGKsn67Hypmdfj4fI75a6EsptbrzXzNxKqokfDT3zItQl4xxA1rFvcDBZOBvfDR9tkqlow5xa3AIA==) seems to be corrupted. Trying again.
npm WARN tarball tarball data for electron-osx-sign@git+ssh://git@github.com/electron/electron-osx-sign.git#02d6ee7bbe4a7f6a2435baef7db84ced5dbc7990 (sha512-4rg0ghhKXHGrSBsE0tlrpNIENbkd0s777WFPeRCySl6pVmwA69Wcmzr79WG2gzGwNzSkiWOhdIoVV/4dBE+6Jw==) seems to be corrupted. Trying again.
npm WARN tarball tarball data for xmldom@git+ssh://git@github.com/xmldom/xmldom.git#c568938641cc1f121cef5b4df80fcfda1e489b6e (sha512-XKfWBSJe4BGKsn67Hypmdfj4fI75a6EsptbrzXzNxKqokfDT3zItQl4xxA1rFvcDBZOBvfDR9tkqlow5xa3AIA==) seems to be corrupted. Trying again.
npm ERR! code EINTEGRITY
npm ERR! sha512-XKfWBSJe4BGKsn67Hypmdfj4fI75a6EsptbrzXzNxKqokfDT3zItQl4xxA1rFvcDBZOBvfDR9tkqlow5xa3AIA== integrity checksum failed when using sha512: wanted sha512-XKfWBSJe4BGKsn67Hypmdfj4fI75a6EsptbrzXzNxKqokfDT3zItQl4xxA1rFvcDBZOBvfDR9tkqlow5xa3AIA== but got sha512-cOF+PgNCQCvHSEYEtJKpRR11b76VLg3BVVCvLBIgeCO4KCQmt3myywGU8fvi3dxudq4UzkL4nIgJajCm1sOsvg==. (32324 bytes)
```

**Comment by McCallumDillon** [comment]
I'll preface this comment with a **_HUGE_** caveat that I am 100% venturing out of my territory here... and I really have no idea what I'm doing/talking about here... _but_... I just moved to 21.04 and in an attempt to upgrade to 1.2.10 ran into the same WARN tarball and ERR flags–  
Someone far more experienced with these matters advised I verify my node version and that lead me to:
`npm version` which informed `node: 16.13.0`, but I none-the-less attempted:
`sudo npm cache clean -f
sudo npm install -g n
sudo n stable`
I'm not sure if forcing the cache clean helped or if any of this is totally unrelated, but I afterward I was able to `sh install-gui.sh` without issue and am back up and running now.
Hope this helps and doesn't confuse/derail– someone who knows what they're talking about please correct me if this is miles off... _just offering to share my experience that got me synced!_

**Comment by wjblanke** [comment]
The above worked for me as well

sudo npm cache clean -f
sudo npm install -g n
sudo n stable

**Comment by luizvital** [comment]
Thanks @McCallumDillon 

These steps really solved the problem here on a Ubuntu 21.04.

Maybe the upgrade of **node** from `12.21.0` to `16.13.0`and **npm** from `7.5.2` to `8.1.0` made the difference
